src/doom/hu_stuff.c

```diff
@@ -217,11 +217,7 @@ char*   mapnames_chex[] =   // Chex Quest names.
     "NEWLEVEL"
 };
 
-// List of names for levels in commercial IWADs
```

Comment:
The 1.9 doom2.exe that comes with Doom II doesn't support Final Doom, so this is a case of version differences.